### ISSUE TITLE
UI editor and disable_canary: secondary_info

### ISSUE THREAD

**Issue by bedson87** [jcwillox/lovelace-canary#18]
Hello! A couple of HA versions ago I made a simple card which I can't change now due to a bug with UI editor. Can anything be done about it? :)

![2021-09-04 14 07 15 192 168 1 11 154549ce6d33](https://user-images.githubusercontent.com/37672068/132093909-8ff4d0ed-ba3d-45ca-a8fc-a1371ee1c788.png)

**Comment by jcwillox** [comment]
Yeah, I've noticed this too, hope there's gonna be a fix for it, it's really annoying. The best way I've found is to use the raw configuration editor, as that won't try and do any weird validation.
